### DISCUSSION TITLE
AzureAD authentication protocol

### DISCUSSION THREAD

**Discussion by rjames4558** [provectus/kafka-ui#3666] [Q&A] [answered]
It appears that SSO with AzureAD is supported, looking at Step 5 of the SSO Guide. Is this using open ID connect , or what authentication protocol? 

https://docs.kafka-ui.provectus.io/configuration/authentication/sso-guide

**Comment by Haarolean** [comment, marked as answer]
Hi, it's OAuth2.
You can copy either example from [here](https://docs.kafka-ui.provectus.io/configuration/authentication/oauth2), just replacing the name of a provider.
